extra_data/sourcedata.py

```diff
@@ -11,10 +11,12 @@
 
 
 class SourceData:
-    """Data for one key in one source
+    """Data for one source
 
     Don't create this directly; get it from ``run[source]``.
     """
+    _device_class = ...
```

Comment:
Why the choice of the ellipsis as a default / unknown value?

Reply:
I wanted to use None to represent 'not available', so I needed something else for 'haven't looked yet'. `...` is kind of a funny option, but I'm happy to use `False` or something if that's clearer.